src/SonarQube.Client/Services/SonarQubeService.cs

```diff
@@ -236,12 +236,7 @@ public Uri GetProjectDashboardUrl(string projectKey)
             allIssuesResult.EnsureSuccess();
 
             return allIssuesResult.Value
-                .Where(x =>
-                {
-                    var resolutionState = SonarQubeIssue.ParseResolutionState(x.Resolution);
-                    return resolutionState == SonarQubeIssueResolutionState.WontFix ||
-                        resolutionState == SonarQubeIssueResolutionState.FalsePositive;
-                })
+                .Where(x => SonarQubeIssue.ParseResolutionState(x.Resolution) != SonarQubeIssueResolutionState.Unresolved)
```

Comment:
We could also do `x => !string.IsEmpty(x.Resolution)`

Reply:
I'd leave it like this: it's clearer what the expected values are.